_posts/2024-04-29-pyos-newsletter-may-2024.md

```diff
@@ -0,0 +1,84 @@
+
+If you’d like to learn more about rdata, we recommend [this guest blog post](https://www.pyopensci.org/blog/read-r-datasets-from-python.html) from Carlos!
+
+### Write a guest blog post for pyOpenSci
+Interested in writing a guest blog post for pyOpenSci? Reach out to [media@pyopensci.org](mailto:media@pyopensci.org) to get started! We’re always looking for volunteers to write about all things open source and open science, whether that’s a package you’ve created or something new you’ve learned in Python, or anything in between!
+
+## <i class="fa-regular fa-handshake"></i> Welcome, ZodiPy
+<figure>
+    <a href="/images/blog/2024/may/zodipy.png">
+    <img src="/images/blog/2024/may/zodipy.png" style="max-width:100%" alt="Screenshot of the ZodiPy GitHub repository, showing the ZodiPy logo and simulated data output.">
+    </a>
+    <figcaption>
+      Welcome, ZodiPy! We're so happy you're here.
+    </figcaption>
+</figure>
+
+[ZodiPy](https://github.com/Cosmoglobe/zodipy) is an open source, [Astropy affiliated](https://www.astropy.org/affiliated/), Python tool for simulating the zodiacal emission that a solar system observer is predicted to see given an interplanetary dust model. ZodiPy attempts to make zodiacal emission simulations more accessible by providing the community with a simple interface to existing models, and is also a new member of the pyOpenSci community!
+
+## <i class="fa-regular fa-message"></i> Listen to Leah at SciPy
+We’ve received the exciting news that our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), will be speaking as well as running a Python packaging workshop at [SciPy](https://www.scipy2024.scipy.org/) this year! We’ll be sharing more about Leah’s work as the conference gets closer, but consider this your reminder to grab your tickets!
+
+## <i class="fa-solid fa-apple-whole"></i> Congratulations on an amazing keynote, Inessa!
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5Bjo6pWj-88?si=fUu5zYPrVCgmPEZF&amp;start=711" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+If you weren’t able to attend PyConDE & PyData Berlin 2024 to hear [Inessa Pawson’s](https://github.com/InessaPawson) keynote, [Tending Open Source Orchards](https://2024.pycon.de/program/7TEYDQ/), never fear! You can [watch her phenomenal talk,](https://www.youtube.com/live/5Bjo6pWj-88?feature=shared&t=711) where she dives into the resilient contributor communities behind every large open source project, and shares her insights on the art and science of fostering resilient open source communities.
+
+## <i class="fa-brands fa-python"></i> Upcoming Python events for scientists
```

Comment:
@kierisi can you please somewhere highlight pyOpenSci at pycon 2024? so you can highlight

1. our maintainers summit with great talks INCLUDING talks by Tracy Teal (board chair) and Carol willing (advisory council). 
2. I am giving a talk on packaging
3. We will lead a 1 day sprint the first day of sprints after the meeting. 

we will have stickers and post cards - so please come say hello and learn more about pyopensci (or something like that). i'm hoping to organize an informal dinner as well one night or gathering even if it's an open space. but maybe they can follow us on socials to keep track of where we are (maybe that is the CTA?)